Themes/default/Profile.template.php

```diff
@@ -3281,4 +3281,136 @@ function template_profile_tfa()
 							</dd>';
 }
 
+/**
+ * Template for initiating and retrieving profile data exports
+ */
+function template_export_profile_data()
+{
+	global $context, $scripturl, $txt;
+
+	$exports_exist = !empty($context['completed_exports']) || !empty($context['active_exports']);
+
+	// The main containing header.
+	echo '
+		<div class="cat_bar">
+			<h3 class="catbg profile_hd">
+				', $txt['export_profile_data'], '
+			</h3>
+		</div>
+		<div class="information">', $context['export_profile_data_desc'], '</div>';
+
+	if (!empty($context['completed_exports']))
+	{
+		echo '
+		<div class="title_bar">
+			<h3 class="titlebg">', $txt['completed_exports'], '</h3>
+		</div>
+		<div class="windowbg noup">
+			<ul class="bbc_list">';
+
+
+		foreach ($context['completed_exports'] as $file)
+			echo '
+				<li>
+					<form action="', $scripturl, '?action=profile;area=getprofiledata;u=', $context['id_member'], '" method="post" accept-charset="', $context['character_set'], '">
+						<a href="', $scripturl, '?action=profile;area=download;u=', $context['id_member'], ';f=', $file['format'], ';t=', $file['dltoken'], '" class="bbc_link">', $file['dlbasename'], '</a> (', $file['size'], ', ', $file['mtime'], ')
```

Comment:
This is where the 'f' is coming from - look at the logic for building the URL here.